ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -4130,3 +4131,130 @@ export const testCollapseBoxPreview = (): Promise<boolean> =>
       ],
     });
   });
+
+export const testTypography = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      await whisper.create({
+        label: 'Are default values displayed correctly?',
+        onClose: () => {
+          console.debug('closed');
+        },
+        components: [
+          {
+            type: WhisperComponentType.Typography,
+            body: 'default = aligin: "inherit", paragraph: false, variant: "body1"',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Center,
+            body: 'Align: Center',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Justify,
+            body: 'Align: Justify',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Left,
+            body: 'Aligin: Left',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Right,
+            body: 'Aligin: Right',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            tooltip: 'tooltip test',
+            body: 'Tooltip',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            variant: 'body1',
+            body: 'body1',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            variant: 'body2',
+            body: 'body2',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            variant: 'button',
+            body: 'button',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            variant: 'caption',
+            body: 'caption',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            variant: 'h1',
+            body: 'h1',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            variant: 'h2',
+            body: 'h2',
+          },
+          {
```

Comment:
there are 3 components with `variant: 'h3'`. not sure if that was intentional or not

Reply:
removed the extras